h2o-core/src/main/java/water/util/MathUtils.java

```diff
@@ -144,13 +147,34 @@ public void reduce(BasicStats bs) {
       }
       return res;
     }
+    
+    public double[] varianceNorm() {
+      return variance(MemoryManager.malloc8d(_mean.length));
+    }
+    // This function calculation variance the normal way
+    private double[] varianceNorm(double[] res) {
```

Comment:
started my own variance calculation since the original variance calculation was actually used by deeplearning even though the values are incorrect.  I think they are using the for different purpose and should really not be called variance.